modules/postman-templating/src/template-client.ts

```diff
@@ -23,10 +23,27 @@ export class TemplateClient {
     this.lineBreak = lineBreak || '<br />'
 
     if (allowedImageSources) {
+      this.validateImageSources(allowedImageSources)
```

Comment:
how about setting the allowedImageSources here?
this.allowedImageSources = allowedImageSources.filter(Boolean)